nifi-nar-bundles/nifi-aws-bundle/nifi-aws-abstract-processors/src/main/java/org/apache/nifi/processors/aws/AbstractAWSProcessor.java

```diff
@@ -319,9 +319,14 @@ protected void initializeRegionAndEndpoint(ProcessContext context) {
                 getLogger().info("Overriding endpoint with {}", urlstr);
 
                 if (urlstr.endsWith(".vpce.amazonaws.com")) {
```

Comment:
Would you extract this into a constant? It is used here and also in row 346.

Reply:
done